src/brc/conf/docker.py

```diff
@@ -41,6 +41,15 @@ def getenv(key, default=None, required=False, split=False):
     # https://github.com/jazzband/django-axes/blob/master/docs/configuration.rst#cache-problems
     'axes_cache': {
         'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
+    },
+    # Cache for BIO removal sync with DRC
+    'drc_sync': {
+        "BACKEND": "django_redis.cache.RedisCache",
+        "LOCATION": "redis://127.0.0.1:6379/2",  # NOTE: watch out for multiple projects using the same cache!
```

Comment:
this cache location should be pulled from the environment, something like `REDIS_CACHE=some-host:6379/0`.

The BRC is deployed with multiple instances - 3 pods, which each have their "own" localhost. There will be one Redis instance in another pod, exposed via a service. The localhost here points to the BRC pod itself, which does not have Redis running.